packages/admin-panel/src/VizBuilderApp/utils/findVizType.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+import { ajvValidate } from '@tupaia/tsutils';
+import { VIZ_TYPES } from '../constants';
+
+export const findVizType = viz => {
```

Comment:
We use this function to determine what viz type to set when opening an existing viz. I considered doing a fuzzy match on type and chart/view type, but I think I prefer this approach as it's a bit simpler and encourages us to have accurate schemas